eng/versioning/external_dependencies.txt

```diff
@@ -173,13 +173,13 @@ cosmos_com.google.guava:guava;25.0-jre
 cosmos_io.dropwizard.metrics:metrics-core;4.1.0
 cosmos_io.dropwizard.metrics:metrics-graphite;4.1.0
 cosmos_io.dropwizard.metrics:metrics-jvm;4.1.0
-cosmos_io.netty:netty-buffer;4.1.45.Final
-cosmos_io.netty:netty-codec-http;4.1.45.Final
-cosmos_io.netty:netty-codec-http2;4.1.45.Final
-cosmos_io.netty:netty-handler;4.1.45.Final
-cosmos_io.netty:netty-handler-proxy;4.1.45.Final
-cosmos_io.netty:netty-tcnative;2.0.27.Final
-cosmos_io.netty:netty-transport-native-epoll;4.1.45.Final
+cosmos_io.netty:netty-buffer;4.1.49.Final
+cosmos_io.netty:netty-codec-http;4.1.49.Final
+cosmos_io.netty:netty-codec-http2;4.1.49.Final
+cosmos_io.netty:netty-handler;4.1.49.Final
+cosmos_io.netty:netty-handler-proxy;4.1.49.Final
+cosmos_io.netty:netty-tcnative;2.0.29.Final
+cosmos_io.netty:netty-transport-native-epoll;4.1.49.Final
```

Comment:
Can these `cosmos_` external dependency version tags be removed in favor of common versions defined above since the versions are the same?

Reply:
Yes, I think these can be removed now. But I am waiting for this PR to go in successfully. 
Once everything looks good for all the SDKs, I will merge them later in a follow up PR.

Reply:
@srnagar  - I have removed these now :)